hack/helmfile.yaml.gotmpl

```diff
@@ -254,13 +254,15 @@ releases:
 
   - name: 'rabbitmq'
     namespace: '{{ .Values.namespace1 }}'
-    chart: '../.local/charts/rabbitmq'
+    chart: 'bitnami/rabbitmq'
+    version: '15.2.0'
```

Comment:
Is there a reason to use `15.2.0` and not `15.2.5` or even a later version? To get RabbitMQ 4.0 (and not 4.1), chart version `15.5.3` might be a good choice.

Reply:
![image](https://github.com/user-attachments/assets/0a3736c6-8c36-45ab-b126-6efbc7867ec9)